sources/packages/backend/libs/sims-db/src/entities/education-program.model.ts

```diff
@@ -66,12 +66,19 @@ export class EducationProgram extends RecordDataModel {
   })
   sabcCode: string;
   /**
-   * Regulatory body ode. Which regulatory body has approved your delivery of this program.
+   * Regulatory body code. Which regulatory body has approved your delivery of this program.
    */
   @Column({
     name: "regulatory_body",
   })
   regulatoryBody: string;
+  /**
+   * Other regulatory body code in case it is not listed in the form.
+   */
+  @Column({
+    name: "other_regulatory_body",
```

Comment:
I can see here that the field was created as `other_regulatory_body` because the existing one is `regulatory_body` in contrast with a similar one for the institutions where they are `regulatoring_body` and `other_regulatoring_body`.
Based on the above, I believe that the DB should be kept as it is but I am just wondering if they should be called the same in the UI at least. Thoughts?

Reply:
I think that "regulatory body" should be used all across the application (UI, Backend, DB) because it is how it is used in the UI.

Reply:
Sorry @andrewsignori-aot , I was not able to exactly follow the context here.

Reply:
@dheepak-aot @ann-aot 
The institution has "regulatory body" as `regulatory_body`/`other_regulatory_body`.
The program has "regulatory body" as `regulatoring_body`/`other_regulatoring_body`

@andrepestana-aot if you follow what you are suggesting we will need to do a migration to execute an "ALTER column" since `regulatory_body` and `regulatoring_body` are existing columns. At this stage, I was asking about maybe having the UI the same, at least.

Reply:
Hi please use regulatory body. I also checked policy manuals and it uses "regulatory". In case the question comes up again, I've captured it on our wiki.

https://github.com/bcgov/SIMS/wiki/Acronyms-&-Terminology#word--spelling-usage
![image](https://github.com/bcgov/SIMS/assets/56935198/f432df88-9398-45f4-bbe8-3cde76d228aa)

Reply:
To be clear, for this PR I would expect only the label change on the UI.
If the team believes that we need to change the program `regulatoring_body/other_regulatoring_body` pair we can maybe have a separate PR to do it.

Reply:
Yes, I got it. Not in this PR but we should change to have only one name all across the application. I thought that we had only "Regulatory body" in the UI but we also have "Regulating body". Then we need to choose between them. :)